### PR TITLE
Prepare release 250.0.0

### PR DESCRIPTION
**Pull request by tarcang**
### What does this PR do?
Update version and author.

### What issues does this PR fix or reference?
@W-14689494@

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ $ npm install -g @salesforce/commerce
 $ sfdx COMMAND
 running command...
 $ sfdx (-v|--version|version)
-@salesforce/commerce/248.1.3 darwin-x64 node-v18.16.1
+@salesforce/commerce/250.0.0 darwin-x64 node-v18.16.1
```
Comment (kalagarraj) [inline README.md:63]: Is there a way to Use the release value from package.json

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ sfdx plugins
...
-* [`sfdx commerce:scratchorg:create [-u <string>] [-a <string>] [-t <string>] [-w <number>] [-y] [-v <string>] [--apiversion <string>] [--json] [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]`](#sfdx-commercescratchorgcreate--u-string--a-string--t-string--w-number--y--v-string---apiversion-string---json---loglevel-tracedebuginfowarnerrorfataltracedebuginfowarnerrorfatal)
+* [`sfdx commerce:scratchorg:create [-u <string>] [-a <string>] [-t <string>] [-w <number>] [-y] [-d <number>] [-v <string>] [--apiversion <string>] [--json] [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]`](#sfdx-commercescratchorgcreate--u-string--a-string--t-string--w-number--y--d-number--v-string---apiversion-string---json---loglevel-tracedebuginfowarnerrorfataltracedebuginfowarnerrorfatal)
```
Comment (kalagarraj) [inline README.md:136]: Release PR should only contain change in semvar.. I recommend moving these additional parameter changes to a separate PR..

File: README.md (hunk 1 above)
Comment (tarcang) [reply README.md:63]: @kalagarraj  We already automated this. README is updated automatically with version changes.

File: README.md (hunk 2 above)
Comment (tarcang) [reply README.md:136]: This is also automated when the version changes...

**Review by kalagarraj — APPROVED** [review]

**Review by Yuanchen-S — APPROVED** [review]
LGTM
